### ISSUE TITLE
Problem with psx games on LibreELEC

### ISSUE THREAD

**Issue by robmcc1983** [zach-morris/plugin.program.iagl#106]
Hi,
I have a problem that only affects psx games.

I download and launch a psx game and everything works fine.

I then exit the game and choose another psx game to play, I allow it to download but when it finishes downloading and launches, it loads the previous game that was downloaded.
It does this every time with psx games, I can reboot the system and it will allow me to play the game I want but then I try another again and it still plays the previous game.

It only affects psx games and I'm using gamestarter add-on, on LibreELEC with a raspberry pi 3.

Thanks

**Comment by zach-morris** [comment]
Interesting.  Can you provide a debug log?
Thanks

**Comment by robmcc1983** [comment]
How do I get a debug log?

**Comment by wonder75** [comment]
I think the PSX game cores autosave the state.
Try entering the in-game emu menu (SELECT+TRIANGLE)
Close the game from there.
After that you should be able to run another game.

**Comment by ITV4Free** [comment]
Works fine on Kodi windows

**Comment by drinfernoo** [comment]
@ITV4Free Yes, this issue affects LibreELEC.

**Comment by zach-morris** [comment]
Could be an autosave thing.  If you disable autosave in Kodi settings does the problem go away?  if so, then I'd call this a Kodi bug (it should only load the autosave based on the game file being played rather than the core)

**Comment by robmcc1983** [comment]
Has nothing to do with kodi.
I use retroarch from the game starter repo and This issue has been there nearly half a year now on libreelec.

**Comment by zach-morris** [comment]
OK, a [debug log](https://wiki.libreelec.tv/how_to/provide_logfile) may shed some light on why.  I can't reproduce this on a Mac or Windows.

**Comment by ITV4Free** [comment]
PSX emulator crashes and i need to somehow add BOIS to it for it to run how do i do that in Kodi

**Comment by drinfernoo** [comment]
@ITV4Free You don't. That's something that needs to be setup at emulator level.

**Comment by robmcc1983** [comment]
Unfortunately I cannot give a debug log as I gave up on all this as it was just giving me problems, libreelec gave me problems and the psx problem was annoying.
I have since upgrade to a small form factor desktop pc, I run Windows 10 now with kodi and launch box for retro gaming. Now using a device with much bigger storage I have all roms stored locally which save the hassle of this addon.
Thanks

**Comment by robmcc1983** [comment]
With people saying this doesn't effect Windows I've just decided to try this addon again on my Windows 10 htpc.

Unfortunately it gives me the same problem as it does on libreelec. 

I have installed the addon, set it up for external emulator (retroarch) then launched crash team racing for psx.

I played for a little while the exited retroarch, back to the addon and selected resident evil for psx as well. 

After it downloaded resident evil, it launched crash team racing. 

It will do this every time unless I exit kodi and reopen kodi then it will load resident evil. 

But the if I exit resident evil and go back to the addon and select crash team racing, it then launches resident evil again. 

As I said I said in the original post months ago it only effects psx this problem and it also appears to be present in both libreelec and windows. 

Thanks

**Comment by robmcc1983** [comment]
It also appears issue has been closed but problem is still there.

**Comment by zach-morris** [comment]
Yes, for me to understand the issue, I need a debug log.  You said you can recreate on Windows.  Can you specify the steps you went through to make that happen?

**Comment by robmcc1983** [comment]
Hi,

All I have to do is open your addon, navigate to Sony PlayStation, select a game I wish to play, when done exit retroarch arch which takes me back to the PlayStation games list. 

Then when I choose a different game to play the previous game will always load even though I've chose a different game to play.

If I then exit kodi and reopen kodi it will let me play a different game. Although it will be stuck in the same loop as above and I'll have to exit kodi everytime I want to play psx game. 

This was exactly the same on libreelec many months ago except I had to reboot libreelec as you can't just exit kodi and relaunch like you can on windows. 

Thanks

**Comment by robmcc1983** [comment]
I also set cache size to 64gb in settings.
That's the only setting I change not sure if that has anything to do with it.

Thanks

**Comment by zach-morris** [comment]
OK, i was able to figure this out. It depended on the alpha order you were actually launching the game.  I believe this is fixed now and will be in the next version
